modules/sns_email_subscription/main.tf

```diff
@@ -19,6 +21,19 @@ resource "aws_cloudformation_stack" "sns_topic" {
     "Email" : {
       "Type" : "String",
       "Description" : "Email for SNS Notifications"
+    },
+    "SlackWebhookUrl": {
```

Comment:
@rjulian so is this a generic webhook, or does it carry a specific payload for slack?  If so, do we really want to be in the business of maintaining webhook payloads in slack?  I was under the impression this was going to be it's own custom lambda that was invoked by our SNS topic?

Reply:
Reached out over chat. It is its own custom lambda  invoked by SNS.

Goal in the long run is to separate all output notification integrations.

Reply:
yup.  opened #92 to track this in the future.